js/admin.discussionpolls.js

```diff
@@ -23,7 +23,7 @@ jQuery(document).ready(function($) {
   var AddQString = $('#DP_NextQuestion').text();
 
   // If there is input in the first question textbox onload, this is an existing poll
-  if ($('#DP_Questions0').val()) {
+  if ($('#DP_Questions01').val()) {
```

Comment:
Did you intend to use '01' here?

I believe the view isn't padding the ID number.

Reply:
Yes. This is part of the fix. The number is not padded, it is actually still `#DP_Questions0`: Gdn_Form uses a static ID `$idCounters` array in newer versions to prevent the same ID from appearing twice (it appends **1**,2,3... which is happening here). This happens, because the form field with the same ID gets generated for the definition list in `PostController_Render_Before`. The change I made in `views/questions.php` is also related, but only works if the poll field is empty.

The more elegant solution would be to have a definition `ExistingPoll` and stop using the HTML fragment in the definition list (`DP_EmptyQuestion`). But that would require rewriting larger parts of the JS and I was just looking for a quick fix to restore all functionality.

Reply:
+1